feature-gate/grpc/featuregate.rs

```diff
@@ -0,0 +1,352 @@
+#[derive(Clone, PartialEq, ::prost::Message)]
+pub struct Feature {
+    #[prost(string, tag = "1")]
+    pub name: std::string::String,
+    #[prost(bool, tag = "2")]
+    pub open: bool,
+}
+#[derive(Clone, PartialEq, ::prost::Message)]
+pub struct FeatureName {
+    #[prost(string, tag = "1")]
+    pub name: std::string::String,
+}
+#[derive(Clone, PartialEq, ::prost::Message)]
+pub struct FeatureAddition {
+    #[prost(string, tag = "1")]
+    pub feature_name: std::string::String,
+    #[prost(sfixed64, tag = "2")]
+    pub guild_id: i64,
```

Comment:
I believe these should be `u64`s technically, even though we use the signed data type in postgres 😕